README.md

```diff
@@ -37,143 +37,160 @@ pre-commit install
 
 ## Basic usage
 
-The package main component is a python object called `SDK` which can be used to create a `Batch` and send it automatically
-to Pasqal APIs using an API token generated in the [user portal](https://portal.pasqal.cloud).
 
-A `Batch` is a group of jobs with the same sequence that will run on the same QPU. For each job of a given batch you must set a value for each variable, if any, defined in your sequence.
-The batch sequence can be generated using [Pulser](https://github.com/pasqal-io/Pulser). See their [documentation](https://pulser.readthedocs.io/en/stable/),
-for more information on how to install the library and create your own sequence.
+### Authentication
 
-Once you have created your sequence, you should serialize it as follows:
+There are several ways to provide a correct authentication using the SDK.
 
 ```python
-# sequence should be a pulser Sequence object
-serialized_sequence = sequence.to_abstract_repr()
+from pasqal_cloud import SDK
+
+project_id = "your_project_id"  # Replace this value with your project_id on the PASQAL platform.
+username = "your_username"  # Replace this value with your username or email on the PASQAL platform.
+password = "your_password"  # Replace this value with your password on the PASQAL platform.
 ```
 
-Once you have serialized your sequence, you can send it with the SDK with the following code
+Ideally, do not write this password in a script but provide in through the command-line or as a secret environment variable.
```

Comment:
Not sure we need this line, people will follow whatever practice they want and it's not particularly relevant